### DISCUSSION TITLE
Github Student Pack "You were last verified as a *blank*...."

### DISCUSSION THREAD

**Discussion by Reigin16** [community/community#114829] [GitHub Education]
### Hi everyone,

I submitted a request on march 29th. After being approved it was first appearing as null. Now it doesn't even say null, as seen in the attached screenshot. Is there some issue? 
![image](https://github.com/community/community/assets/128984477/f241c7bf-74e3-4edb-9333-1cff5b3e4a0c)

**Comment by RkBeau** [comment]
Same.  Precisely the same.

**Comment by samus-aran** [comment]
Hi @RkBeau  👋

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [You were last verified as a "null" on...](https://github.com/orgs/community/discussions/111352#null-last-verified-on-date)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
